src/components/10-atoms/input-text/README.md

```diff
@@ -84,9 +84,9 @@ The boolean attribute `required` visualizes an element that must obligatorily be
 
 String-valued `placeholder`specifies the placeholder text shown when the element is empty.
 
-### valid
+### invalid
 
-The boolean attribute `valid` serves to indicate the validity of the element (default: `true`).
+The boolean attribute `invalid` serves to indicate the validity of the element (default: `false`). if `false` it sets the element into a visual error state.
 
 ### validation
```

Comment:
validation
The boolean attribute validation shows an animated checkmark to the right of the input-text, if valid is true (default: false). 

-> Better name might be `animation`

Reply:
same here:
<img width="765" alt="Bildschirmfoto 2019-07-02 um 11 43 25" src="https://user-images.githubusercontent.com/275235/60502771-b079c380-9cbe-11e9-88fc-cfdc2bc76d43.png">

Reply:
fixes in 2238ff5